main.rb

```diff
@@ -0,0 +1,21 @@
+require './factory'
+
+def main
+  factory = Factory.new
+
+  loop do
+    choice = factory.run
+    case choice
+    when 1..8
+      factory.process(choice)
+    when 9
+      puts 'App quitting gracefully..'
+      break
+    else
+      puts 'INVALID OPTION'
+    end
+    factory.separator
+  end
+end
+
```

Comment:
- [ ] When I want to create a new book, I am getting this error
![library](https://user-images.githubusercontent.com/61285247/186895293-4ace87f6-5662-473b-a11d-2ee4ecdda174.png)
Please kindly fix that.